controllers/setup.go

```diff
@@ -50,13 +50,15 @@ func (r *KymaReconciler) SetupWithManager(mgr ctrl.Manager,
 	options controller.Options, settings SetupUpSetting,
 ) error {
 	generationChangedPredicate := predicate.GenerationChangedPredicate{}
+	labelChangedPredicate := predicate.LabelChangedPredicate{}
+	predicates := predicate.Or(generationChangedPredicate, labelChangedPredicate)
```

Comment:
you can avoid declaring those two predicates but directly write in predicate.Or()